src/integrationTest/java/uk/gov/hmcts/reform/roleassignment/controller/WelcomeControllerIntegrationTest.java

```diff
@@ -57,5 +60,26 @@ public void welComeAPITest() throws Exception {
         assertEquals(
             "Service is unavailable:", 200, 200);
```

Comment:
the message is looking wrong for 200 status

Reply:
Message updated.